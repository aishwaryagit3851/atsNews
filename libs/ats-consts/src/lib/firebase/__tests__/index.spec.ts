import { FCM_BASE_URL } from '..';

describe('ATS CONSTS for - firebase', () => {
  it('Match consts to snapshot', () => {
    expect(FCM_BASE_URL).toMatchSnapshot();
  });
});
