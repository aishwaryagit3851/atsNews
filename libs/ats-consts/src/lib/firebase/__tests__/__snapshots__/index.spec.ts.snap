// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ATS CONSTS for - firebase Match consts to snapshot 1`] = `"https://fcm.googleapis.com/fcm/send"`;
